drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -909,37 +606,31 @@ static void timeout_work_fn(struct k_work *work)
 	struct gps_event evt = {
```

Comment:
There is an idiom for obtaining the data which takes into consideration the delayable nature of the work queue item. Please see the [Delayable Work](
https://docs.zephyrproject.org/latest/reference/kernel/threads/workqueue.html?highlight=container_of#delayable-work) API documentation.

Reply:
I'll fix this.